www/css/ionic.app.css

```diff
@@ -9277,6 +9277,10 @@ body * {
   .sign-in :-ms-input-placeholder {
     color: #f2f2f2; }
 
+.trade #tradeContractChart {
```

Comment:
I didn't. It was the gulp watch. Ok, I will :)

On Tue, Dec 15, 2015 at 1:11 PM, Massih hazrati <notifications@github.com>
wrote:

> In www/css/ionic.app.css
> <https://github.com/binary-com/mobile/pull/135#discussion_r47615113>:
>
> > @@ -9277,6 +9277,10 @@ body * {
> >    .sign-in :-ms-input-placeholder {
> >      color: #f2f2f2; }
> >
> > +.trade #tradeContractChart {
>
> Do not update .css files directly, please revert the commit and apply
> changes to scss files
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/binary-com/mobile/pull/135/files#r47615113>.
>